zend_abstract_interface/interceptor/php7/interceptor.c

```diff
@@ -0,0 +1,1108 @@
+#include "interceptor.h"
+#include "../../hook/hook.h"
+#include "../../hook/table.h"
+#include "zend_vm.h"
+#include <ext/standard/basic_functions.h>
+#include <Zend/zend_exceptions.h>
+#include <Zend/zend_generators.h>
+#include <pthread.h>
+
+#if PHP_VERSION_ID < 70100
+static ZEND_FUNCTION(pass)
+{
+    (void)execute_data, (void)return_value;
+}
+
+// HACK: define a name for XDebug compatibility...
+static const struct __attribute__((__packed__)) { zend_string str; char value[]; } zend_pass_function_name = {
+        .str.gc.refcount = 2,
+        .str.gc.u.v.type = IS_STRING,
+        .str.gc.u.v.flags = IS_STR_INTERNED,
+        .str.len = 11,
+        .str.h = 0xc0ff0fab5cdbc6c4,
```

Comment:
Uh, some comments here, please?

Reply:
it's just the hash, I manually evaluated it, C has no constexpr :-D

Reply:
The function name is `zend_pass}`?